src/sfx.cpp

```diff
@@ -564,6 +564,24 @@ void sfx_play_wm_footstep(int player_model_id, int player_walkmap_type)
 	}
 }
 
+void sfx_process_wm_highwind(bool is_highwind_motor_on)
+{
+	constexpr int default_sfx_id = 159;
+
+	if (use_external_sfx)
+	{
+		if(is_highwind_motor_on)
+			ff7_sfx_set_panning_on_channel(32, 7);
+		else
+		 	ff7_sfx_set_panning_on_channel(0, 7);
+
+		char track_name[64];
+		sprintf(track_name, "sfx_highwind");
+		if(!nxAudioEngine.isSFXPlaying(7))
+			nxAudioEngine.playSFX(track_name, default_sfx_id, 7, 0.0f);
+	}
```

Comment:
Can you please add this block here?
```cpp
else
{
  common_externals.play_sfx(default_sfx_id);
}
```

I'd also move the block
```cpp
if(is_highwind_motor_on)
  ff7_sfx_set_panning_on_channel(32, 7);
else
  ff7_sfx_set_panning_on_channel(0, 7);
```

Out side the if so it's common for both cases. LMK what you think.

Reply:
I didn't want to add sound for non-external sfx because I don't think there is a good sound. Tiny bronco one is kinda bad for highwind. I also need to find the original id for that sfx

Reply:
Cannot modify panning on non-external sfx even by using `common_externals.set_sfx_panning_on_channel` function. Probably the channel value is wrong.

Reply:
Yes, channel 7 doesn't exist in vanilla. I don't remember which one is used for bronco so you have to use the same. I think it's 5 but not 100% sure.

Regarding the else block, allright, if it's intended fine by me.